.github/workflows/statuses.yml

```
@@ -0,0 +1,32 @@
+name: Update Statuses Board
```

In general, we prefer to use sentence case for most headings: https://primer.style/design/foundations/content#top-10-rules
```suggestion
name: Update status board
```